get_github_coderabbit_review/scripts/test_uvx_installation.py

```
@@ -117,6 +117,9 @@ def test_package_installation(self) -> bool:
         """Test package installation with uvx."""
         print("📦 Testing package installation...")
 
+        # Test local package installation
+        project_root = Path(__file__).parent.parent
```

Ruff (0.13.1)

Local variable `project_root` is assigned to but never used

Remove assignment to unused variable `project_root`

(F841)